src/honey/sql.cljc

```diff
@@ -214,11 +214,13 @@
      (fn [fk _] (param-value (fk)))}))
 
 (defn- format-var [x & [opts]]
-  (let [c (name-_ x)]
+  (let [c (name x)]
     (cond (= \% (first c))
-          (let [[f & args] (str/split (subs c 1) #"\.")]
-            ;; TODO: this does not quote arguments -- does that matter?
-            [(str (upper-case f) "(" (str/join "," args) ")")])
+          (let [[f & args] (str/split (subs c 1) #"\.")
+                quoted-args (->> args
+                                 (map keyword)
+                                 (map format-entity))]
```

Comment:
`opts` needs to be passed through into `format-entity` here (as is done for the call near the end of this function and, ideally, tests need to also check that `:aliased` and `:drop-ns` options work correctly when passed through `format-var`. That also means that tests with qualified column names are needed, i.e., `f-o/b-r` as a column argument.

Reply:
Looking over the contexts that would set `:aliased` or `:drop-ns` to be `true`, I'm pretty certain that none of those would be legal for a function invocation so writing tests would be hard (short of actually testing the private function directly, which I try to avoid). I think just changing this to `(map #(format-entity % opts))` would be fine although I would prefer this expression to be `(map #(format-entity (keyword %) opts) args)` since it avoids the double-`map` and it's not really worth using the transduce arities here because there's likely only one or two arguments.